tests/v2/validation/provisioning/registries/registry_test.go

```diff
@@ -317,45 +375,59 @@ func (rt *RegistryTestSuite) TestRegistriesRKE2() {
 	subSession := session.NewSession()
 	defer subSession.Cleanup()
 
-	subClient, err := rt.client.WithSession(subSession)
+	subClient, err := rt.standardUserClient.WithSession(subSession)
 	require.NoError(rt.T(), err)
+
+	if rt.provisioningConfig.RKE2KubernetesVersions == nil {
+		rke2Versions, err := kubernetesversions.ListRKE2AllVersions(rt.client)
+		require.NoError(rt.T(), err)
+
+		rt.provisioningConfig.RKE2KubernetesVersions = rke2Versions
+	}
+
 	tests := []struct {
 		registry string
 		name     string
 	}{
-		{rt.privateRegistriesNoAuth[0].URL, "RKE2 Registry No Auth"},
-		{rt.privateRegistriesAuth[0].URL, "RKE2 Registry Auth"},
+		{rt.privateRegistriesNoAuth[0].URL, "RKE2 Registry No Auth "},
+		{rt.privateRegistriesAuth[0].URL, "RKE2 Registry Auth "},
```

Comment:
What's the reason for the space at the end of the string?

Reply:
we append the k8sVersion later, so putting a space here for readability. I could add it later, but its a little prettier to have it here vs. a ' ' on line 398